src/main/java/com/stackedsuccess/controllers/GameBoardController.java

```diff
@@ -318,4 +325,12 @@ public void gameOver() throws IOException {
 
         System.exit(0);
     }
+
+    /**
+     * Method for when pressing the pause button, this will pause the current game instance
+     *
+     * @throws IOException
+     */
+    @FXML
+    public void onEscapeButton(ActionEvent event) throws IOException {}
```

Comment:
Please delete this method. OnKeyPressed links to the GameInstance method handleInput() which already deals with pressing the ESC button to pause.